### PR TITLE
Add index provider documentation

### PR DESCRIPTION
**Pull request by masih**
Add the index provider documentation.

### CODE REVIEW

**Review by johnnymatthews — APPROVED** [review]

**Comment by masih** [comment]
Thanks @johnnymatthews 🍻

File: content/en/docs/storage-providers/index-provider.md [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+
+{{< alert icon="tip" >}}
+Note that if you have split your markets subsystem, you will need to specify `--call-on-market`.
+{{< /alert >}}
+
+## Configuration
+
+You can adjust the values under the `IndexProvider` session in the `config.toml` of your market process to configure indexes' announcement to the indexer.
+
+If the session doesn't exist, you can manually add it:
```
Comment (magik6k) [inline content/en/docs/storage-providers/index-provider.md:173]: `s/session/section`?

Comment (masih) [reply content/en/docs/storage-providers/index-provider.md:173]: Thank you for pointing this out.

My spelling mistakes will outlive me; I am sure.

Comment (masih) [reply content/en/docs/storage-providers/index-provider.md:173]: Fix added here: https://github.com/filecoin-project/lotus-docs/pull/94

**Pull request merged**